### ISSUE TITLE
spec logs spec ids instead of useful info

### ISSUE THREAD

**Issue by jmrodri** [openshift/ansible-service-broker#297]
```
[2017-07-22T17:24:13.402Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:13.595Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/jenkins-apb into Spec
[2017-07-22T17:24:13.63Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:13.837Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/pyzip-demo-db-apb into Spec
[2017-07-22T17:24:13.879Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:14.144Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/rhscl-postgresql-apb into Spec
[2017-07-22T17:24:14.182Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:14.336Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/pyzip-demo-apb into Spec
[2017-07-22T17:24:14.383Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:14.757Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/mediawiki123-apb into Spec
[2017-07-22T17:24:14.792Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:15.166Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/wordpress-ha-apb into Spec
[2017-07-22T17:24:15.204Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:15.339Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/hello-world-apb into Spec
[2017-07-22T17:24:15.377Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:15.576Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/rhscl-mariadb-apb into Spec
[2017-07-22T17:24:15.621Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:15.988Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/rocketchat-apb into Spec
[2017-07-22T17:24:16.023Z] [DEBUG] Registry::imageToSpec
[2017-07-22T17:24:16.375Z] [DEBUG] Successfully converted Image ansibleplaybookbundle/etherpad-apb into Spec
[2017-07-22T17:24:16.419Z] [DEBUG] specs -> [0xc4204a9ad0 0xc4204a9b80 0xc4204a9c30 0xc4204a9ce0 0xc4204a9d90 0xc42070
6a50 0xc420706bb0 0xc420852000 0xc420706d10 0xc420706dc0]
```

**Comment by eriknelson** [comment]
Think I killed this in #298

**Comment by jmrodri** [comment]
Fixed by PR #298
